src/frontends/pytorch/src/op_table.cpp

```diff
@@ -50,6 +50,8 @@ OP_CONVERTER(translate_to);
 OP_CONVERTER(translate_transpose);
 OP_CONVERTER(translate_tuple_construct);
 OP_CONVERTER(translate_view);
+OP_CONVERTER(translate_addcmul);
```

Comment:
Could you please sort it alphabetically?

Reply:
done